revgen/app.go

```diff
@@ -116,21 +115,11 @@ VERSION: {{.Version}}
 func findRootDir(configFileName string) *string {
 	currDir, err := os.Getwd()
 	check(err)
-	out, err := runCmd("git rev-parse --show-toplevel", &currDir)
+	_, err = runCmd("git rev-parse --show-toplevel", &currDir)
 	check(err)
-	rootDir := strings.TrimSpace(out)
-
-	for {
```

Comment:
the idea of this loop is to look for the config file starting in the current directory and going up a directory until we reach the root directory of the repo. I think this change breaks that.